src/main/java/com/squareup/osstrich/JavadocPublisher.java

```diff
@@ -128,7 +131,26 @@ private void writeIndexFiles(String groupId, Multimap<String, Artifact> artifact
 
       File indexHtml = new File(directory + "/" + majorVersion + "/index.html");
       Files.write(html, indexHtml, UTF_8);
-      gitAdd(indexHtml);
+      //gitAdd(indexHtml);
+    }
+  }
+
+  private String findRelativePath(File directory, String artifactId) {
+    if (!directory.isDirectory()) {
+      throw new IllegalArgumentException(directory + " is not a directory!");
+    }
+    File[] subfiles = Objects.requireNonNull(directory.listFiles());
```

Comment:
Instead of listing all files and looping, you could use `new File(directory, "index.html")` and then check if it exists or is a directory.

Reply:
d'oh, yes that's a much simpler check. Rebased and switched to this in f2b77cc